backend/src/main/java/com/bakdata/conquery/io/storage/MetaStorage.java

```diff
@@ -196,4 +197,24 @@ public void addFormConfig(FormConfig formConfig) {
 	public MutableInjectableValues inject(MutableInjectableValues values) {
 		return values.add(MetaStorage.class, this);
 	}
+
+	public <ID extends Id<VALUE>, VALUE extends Identifiable<?>> VALUE get(ID id) {

```

Comment:
Diese get Methoden in den Storages imitieren einfach das ehemalöige getOptional der CentralRegistry.